Classes/CodingDojo.sc

```diff
@@ -1,19 +1,21 @@
 CodingDojo {
-	var <username, <password, <serveraddress, <serverport;
+	var <username, <password, <serveraddress, <serverport, <groupName, <groupPassword;
```

Comment:
groupName and password can just live in oscrouter, copies here can go out of sync.